scss/_patterns_side-navigation.scss

```diff
@@ -432,12 +432,14 @@
     }
 
     &:hover,
-    &.is-active {
+    &.is-active,
+    &[aria-current] {
```

Comment:
Should we specify `[aria-current="page"]` as you could set `[aria-current="false"]` to indicate does not represent item within a set.

Reply:
Ah, good point. Didn't know you can set it to "false".